docs/representer/MENTORING.md

```diff
@@ -0,0 +1,13 @@
+# Mentoring
```

Comment:
We could also consider this document to be a general, track-specific document on mentoring which then could also be used on the general mentoring page. Thoughts?